config/samples/disk_formats/create-image.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
-1: Can you rename this script? It would be misleading a `create-image.sh` script (that does a different validation) in the openstackclient pod.